### PR TITLE
Added Ratings Component

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Ratings Component

### CODE REVIEW

File: styles/ratings.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.rating-wrapper label:before {
+  content: "\2729";
+  position: absolute;
+  display: inline-block;
+  top: 0;
+  right: 0;
+  width: 50rem;
+  font-size: 20px;
```
Comment (Shivanshsharma01) [inline styles/ratings.css]: can use relative units above 16px

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Suggested change can be taken care of later

**Review by ZuberDunge — APPROVED** [review]
Everything looks fine. Good to go from my side.

File: styles/ratings.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/ratings.css]: Thanks for bringing it to notice, I have changed it now

**Pull request merged**